application/language/english/ip_lang.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 $lang['cldr'] = 'en';
+//Day of the week start. 0 (Sunday) to 6 (Saturday)
+$lang['week_start'] = 0;
```

Comment:
Is this really language specific? I think it's more a religious thing, which means this should be placed in the settings.